src/test/java/org/roaringbitmap/buffer/TestUtil.java

```diff
@@ -0,0 +1,65 @@
+package org.roaringbitmap.buffer;
+
+
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.nio.Buffer;
+import java.nio.ByteBuffer;
+import java.nio.LongBuffer;
+import java.nio.ShortBuffer;
+import java.util.Arrays;
+
+public class TestUtil {
+
+    @Test(expected = IllegalArgumentException.class)
+    public void testFillArrayANDNOT() {
+        LongBuffer data1 = LongBuffer.wrap(new long[]{1, 2, 4, 8, 16});
+        LongBuffer data2 = LongBuffer.wrap(new long[]{2, 1, 3, 7, 15});
+        LongBuffer data3 = LongBuffer.wrap(new long[]{2, 1, 3, 7});
+        short[] content = new short[5];
+        short[] content2 = new short[5];
+        short[] result = {0, 65, 130, 195, 260};
+        BufferUtil.fillArrayANDNOT(content, data1, data2);
+        Assert.assertTrue(Arrays.equals(content, result));
+        BufferUtil.fillArrayANDNOT(content2, data1, data3);
```

Comment:
I would move the last `BufferUtil.fillArrayANDNOT` in another test that would have the expected exception annotation, just to make the test more robust.
Indeed, if the first `BufferUtil.fillArrayANDNOT`is throwing the `IllegalArgumentException` because of a new regression, then the test will pass and the regression will go unnoticed.